### DISCUSSION TITLE
How to disable specific blocks in WordPress

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#139] [Topic Ideas]
"How do I disable the Spacer block?"
"How do I disable certain blocks based on the role of the user?"
"How do I disable certain blocks on posts but not on pages?"

These questions seem to come up often, and there is no comprehensive guide on how to do this and what the current limitations are around controlling blocks. This article would explore this topic and provide examples of how to achieve common use cases.

**Comment by bph** [comment]
The topic has been approved. The discussion is now locked,
Please continue the conversation on the issue
- #148

**Comment by ndiego** [comment]
Post is published: https://developer.wordpress.org/news/2024/01/29/how-to-disable-specific-blocks-in-wordpress/
